mr/src/test/java/org/apache/mahout/classifier/df/tools/VisualizerTest.java

```diff
@@ -17,9 +17,7 @@
 
 package org.apache.mahout.classifier.df.tools;
 
-import java.util.List;
-import java.util.Random;
-
+import com.google.common.collect.Lists;
```

Comment:
I am somehow not able to make the changes get reflected in this PR. Let me close this and open a new PR fresh.